docs/layers.md

```diff
@@ -739,6 +739,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 
 #### Place properties (common optional):
 
+* `kind:xx`: alternate `kind` according to the viewpoint `XX`. If you want to show features according to a particular viewpoint, then use `kind:xx` if it exists and fall back to `kind` if not. The range of values is the same as for `kind`. Note that the viewpoints are either lower-case [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) codes or the pseudo-code `iso`.
```

Comment:
I think the kind for all cities is `locality`, and we don't alter that for viewpoint. I think it's just the `country_capital:xx` and `region_capital:xx` (which I think are documented a few lines down).

Reply:
Fixed in 09d49598bc0b37ebacaa59958c93a1f5c0cca7fa.